torch/csrc/jit/passes/mkldnn_rewrite.cpp

```diff
@@ -6,13 +6,50 @@
 #include <torch/csrc/jit/passes/dead_code_elimination.h>
 #include <torch/csrc/jit/passes/graph_rewrite_helper.h>
 #include <torch/csrc/jit/passes/mkldnn_rewrite.h>
+#include <torch/csrc/jit/passes/mkldnn_rewrite_helper.h>
 #include <torch/csrc/jit/tensorexpr/kernel.h>
 
 namespace torch {
 namespace jit {
 
 #if AT_MKLDNN_ENABLED()
 
+namespace mkldnn {
+
+const std::vector<std::string> zero_scalar_operand =
+    std::vector<std::string>({});
+const std::string algorithm_indicator = std::string("%algorithm");
```

Comment:
This variable is shadowed by the argument in the function below. Can we delete it?

Reply:
yes,  delete it.